jsx/CSSGrid.js

```diff
@@ -16,38 +17,108 @@ import Card from 'Card';
  * @return {object} - A React component for a CSS grid of cards
  */
 function CSSGrid(props) {
-  const grid = {
-    display: 'grid',
-    gridTemplateColumns: '33% 33% 33%',
-    gridAutoFlow: 'row dense',
-    gridRowGap: '1em',
-    rowGap: '1em',
-  };
-
-  let cards = props.Cards.map((value, idx) => {
-    let cardID = 'card' + idx;
-
-    let style = {};
-    if (value.Width) {
-      style.gridColumnEnd = 'span ' + value.Width;
-    }
-    if (value.Height) {
-      style.gridRowEnd = 'span ' + value.Height;
+    const cardsRef = useRef(null);
```

Comment:
The `cardsRef` doesn't seem to be used.

Reply:
I'm not sure what you mean.. it's used a couple places according to ctrl-f?

Reply:
@driusan never mind.